timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,141 @@
+# VPC Peering in Timescale Forge
+
+Timescale Forge allows you to create a private network peering connection between
+your cloud VPC and the Timescale Forge cloud infrastructure where your Timescale
+services are hosted.
+
+VPC peering isolates your Timescale Forge services ensuring that they are only
+accessible via your peered VPC, providing a greater security model due to reduced
+attack vector.
+
+The Timescale Forge console makes creating and configuring your VPC peering connections
+simple, providing controls for adding and removing VPC peering connections, migrating
+services to and from VPCs, and creating new services with VPC peering attachments.
+
+## Requirements and limitations
+To use Timescale Forge VPC peering, you need your own cloud VPC, where your
+applications and infrastructure are already running.
+
+If you do not have administrative access to your cloud provider account, you will need
+to work with someone from your team with sufficient permissions to:
+
+- accept VPC peering requests,
+- configure route table rules,
+- configure security group and firewall rules.
+
+<highlight type="tip">
+The Timescale Forge VPC peering feature is currently limited to AWS VPC peering only.
+</highlight>
+
+<highlight type="tip">
+Timescale Forge defaults to a limit of 3 VPCs per Forge project. If more VPCs are needed,
+you may contact support to request a quota increase.
+</highlight>
+
+<highlight type="warning">
+Once you have attached your Timescale Forge service to a VPC, it will no longer be accessible
+via the public internet. It will only be accessible via your AWS VPC which has been peered
+with your Timescale Forge VPC.
+
+Timescale Forge uses a different DNS name for a Timescale service once it has been attached
+to a VPC. This means that you will need to update your connection string if migrating a service
+from the public internet into a VPC, or vice-versa.
+</highlight>
```

Comment:
This is a long string of admonitions. Perhaps we could leave the warnings, but move the tips back out into paragraphs? I've found that when you have more than two in a row, they tend to *all* get ignored.

Reply:
Ok, I'm actually going to move this last paragraph of the warning to a different section based on some review that Luke provided. Thanks @Loquacity !